UnitySDK/Assets/ML-Agents/Examples/3DBall/Scenes/3DBall.unity

```diff
@@ -851,11 +851,10 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: eb15e3c3d55e54abaafb74c635b6a458, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+  trainInEditor: 0
```

Comment:
Is this used anywhere? Don't see it referenced in the code.

Reply:
Nope, not used

Reply:
It's not just here, it's in (almost?) all the other modified assets too.